ingestion/src/metadata/ingestion/api/topology_runner.py

```diff
@@ -245,7 +226,7 @@ def _(
         lineage has been properly drawn. We'll skip the process for now.
         """
         yield entity_request
-        self.update_context(stage=stage, entity=right)
+        self.update_context(stage=stage, entity_name=right)
```

Comment:
Maybe we can update this call. Not sure why we are passing the full `right` to the `entity_name`